client/package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
+    "up": "docker-compose --env-file ../docker-compose.env -f ../docker-compose.yml up",
```

Comment:
after you merge in dev, can you add these commands to the readme?

Reply:
Completed, thank you!